### ISSUE TITLE
Error Nanopolish consensus 

### ISSUE THREAD

**Issue by livmoe** [jts/nanopolish#549]
I have ran nanopolish - -version 0.11.0 (installed with conda) on Canu assembled data. 

nanopolish index -v -d fast5_pass/ FAK37076_cat.fastq 
[readdb] indexing fast5_pass/
[readdb] num reads: 448252, num reads with path to fast5: 448252

minimap2 -ax map-ont -t 8 assembly_canu/contigs.canu.fasta FAK37076_cat.fastq | samtools sort -o canu.contigs.sorted.bam -T reads.tmp 

nanopolish variants --consensus -o canu.polishedContigs_chr.vcf -r FAK37076_cat.fastq -w tig00000001 -b canu.contigs.sorted.bam -g assembly_canu/contigs.canu.fasta 

And I get the error:

nanopolish: src/alignment/nanopolish_alignment_db.cpp:267: std::vector<Variant> AlignmentDB::get_variants_in_region(const string&, int, int, double, int) const: Assertion `stop_position >= start_position' failed.

I saw the post #403 , but this is not exactly the same error. Could this also be a indexing problem?

**Comment by jts** [comment]
You possibly ran out of memory. Can you try a smaller polishing window, or using the `nanopolish_makerange.py` script described in the README?

**Comment by livmoe** [comment]
Thanks for the fast response!

I tried to use the nanopolish_makerange.py as suggested in readme. 

python nanopolish_makerange.py assembly_canu/pantoea.contigs.canu.fasta | parallel --results nanopolish.results -P 8     nanopolish variants --consensus -o polished.{1}.vcf -w {1} -r FAK37076_cat.fastq -b canu.contigs.sorted.bam -g assembly_canu/pantoea.contigs.canu.fasta -t 8 --min-candidate-frequency 0.1
python: can't open file 'nanopolish_makerange.py': [Errno 2] No such file or directory

But I gave it a new chance now without explicitly calling python in front, and it seems to be running now.

**Comment by livmoe** [comment]
Thanks for the fast response!

I tried to use the nanopolish_makerange.py as suggested in readme.

python nanopolish_makerange.py assembly_canu/contigs.canu.fasta | parallel --results nanopolish.results -P 8 nanopolish variants --consensus -o polished.{1}.vcf -w {1} -r FAK37076_cat.fastq -b canu.contigs.sorted.bam -g assembly_canu/pantoea.contigs.canu.fasta -t 8 --min-candidate-frequency 0.1
python: can't open file 'nanopolish_makerange.py': [Errno 2] No such file or directory

But I gave it a new chance now without explicitly calling python in front, and it seems to be running now.
